app/Models/Post.php

```diff
@@ -177,6 +177,43 @@ public function toBlinkPayload()
         ];
     }
 
+    /**
+     * Transform the post model for Quasar.
+     *
+     * @return array
+     */
+    public function toQuasarPayload()
+    {
+        return [
+            'id' => $this->id,
+            'signup_id' => $this->signup_id,
+            'northstar_id' => $this->northstar_id,
+            'type' => $this->type,
+            'action' => $this->action,
+            'quantity' => $this->quantity,
+            // Add cache-busting query string to urls to make sure we get the
+            // most recent version of the image.
+            // @NOTE - Remove if we get rid of rotation.
+            'media' => [
+                'url' => $this->getMediaUrl(),
+                'caption' => $this->caption,
+            ],
+            'tags' => $this->tagSlugs(),
+            'reactions' => [
+                'total' => isset($this->reactions_count) ? $this->reactions_count : null,
```

Comment:
Let's remove this `reactions` property, since this `reactions_count` won't be set here. (And can be better inferred with a join or whatnot in the warehouse itself)>